### DISCUSSION TITLE
refactor branch prefix 추가

### DISCUSSION THREAD

**Discussion by ljw0096** [prgrms-fe-devcourse/FEDC2_TimeDivider_Root#31] [1. 비개발]
이번에 각자 개발한 페이지를 리팩터링하고 이후 작업을 진행하기로 했는데 기존 branch prefix 는 feature, hotfix 두개밖에 존재하지 않습니다.

refactor prefix를 룰에 추가하면 어떨까요?

**Comment by dahye1013** [comment]
이 discussion을 못보고 진행해서 저는 feature-refactor 이라고 prefix 지어서 진행했네요 핳...!
좋은 생각인것 같아요! 다음 브런치부터는 그렇게 파도록 하겠습니다 :)
